<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>440</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>85</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>85</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>85</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>85</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>85</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>85</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>85</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>85</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>85</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>RB Package</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 85, 255);
font: 75 10pt &quot;나눔바른고딕&quot;;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
     <pointsize>10</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="Name">
          <property name="minimumSize">
           <size>
            <width>618</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>52</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 85, 255);
font: 52pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>RB Package</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="Editor">
            <property name="minimumSize">
             <size>
              <width>325</width>
              <height>40</height>
             </size>
            </property>
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>220</red>
                  <green>220</green>
                  <blue>220</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>220</red>
                  <green>220</green>
                  <blue>220</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Window">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>220</red>
                  <green>220</green>
                  <blue>220</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>220</red>
                  <green>220</green>
                  <blue>220</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>220</red>
                  <green>220</green>
                  <blue>220</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Window">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>220</red>
                  <green>220</green>
                  <blue>220</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>220</red>
                  <green>220</green>
                  <blue>220</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>220</red>
                  <green>220</green>
                  <blue>220</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Window">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>220</red>
                  <green>220</green>
                  <blue>220</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 10pt &quot;나눔스퀘어라운드 Bold&quot;;
background-color: rgb(220, 220, 220);</string>
            </property>
            <property name="text">
             <string>RB.Editor (보안 에디터)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="InTBM">
            <property name="minimumSize">
             <size>
              <width>325</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 10pt &quot;나눔스퀘어라운드 Bold&quot;;
background-color: rgb(220, 220, 220);</string>
            </property>
            <property name="text">
             <string>RB.InTBM (인터넷 북마크)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Blogs">
            <property name="minimumSize">
             <size>
              <width>325</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 10pt &quot;나눔스퀘어라운드 Bold&quot;;
background-color: rgb(220, 220, 220);</string>
            </property>
            <property name="text">
             <string>RB.Blogs (빨파의 블로그)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Links">
            <property name="minimumSize">
             <size>
              <width>325</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 10pt &quot;나눔스퀘어라운드 Bold&quot;;
background-color: rgb(220, 220, 220);</string>
            </property>
            <property name="text">
             <string>RB.Links (파일 / 폴더 바로가기)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="InTman">
            <property name="minimumSize">
             <size>
              <width>325</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 10pt &quot;나눔스퀘어라운드 Bold&quot;;
background-color: rgb(220, 220, 220);</string>
            </property>
            <property name="text">
             <string>인터넷 관리자 페이지</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QTabWidget" name="Maintab">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>275</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_1">
           <attribute name="title">
            <string>현재 시간</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="1">
             <widget class="QWebEngineView" name="TimeNow" native="true">
              <property name="url" stdset="0">
               <url>
                <string>https://vclock.kr/embed/time/</string>
               </url>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>공지</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QTextEdit" name="textEdit">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'나눔바른고딕'; font-size:10pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;&amp;lt; 공지사항 &amp;gt;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;테스트 버전입니다.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::NoTextInteraction</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
selection-color: rgb(0, 255, 0);
</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>메뉴</string>
    </property>
    <addaction name="action_Options"/>
    <addaction name="action_Hidden"/>
    <addaction name="separator"/>
    <addaction name="action_Info"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Info">
   <property name="text">
    <string>정보</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="action_Options">
   <property name="text">
    <string>설정</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="action_Hidden">
   <property name="text">
    <string>고급 설정</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>InTBM</tabstop>
  <tabstop>Blogs</tabstop>
  <tabstop>Links</tabstop>
  <tabstop>InTman</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
